usr/lib/hypnotix/common.py

```diff
@@ -161,7 +161,7 @@ def get_playlist(self, provider, refresh=False) -> bool:
                         # Set downloaded size
                         downloaded_bytes = 0
                         # Get total playlist byte size
-                        total_content_size = int(response.headers['content-length'])
+                        total_content_size = int(response.headers.get('content-length', 0))
```

Comment:
Have you considered a default of 1 (or [15](https://github.com/linuxmint/hypnotix/blob/067e90a14a1c28de5bd8ca4e2e9adb2f5f98c273/usr/lib/hypnotix/common.py#L201)) instead so at least empty files of failed downloads are not left dangling?

https://github.com/linuxmint/hypnotix/blob/067e90a14a1c28de5bd8ca4e2e9adb2f5f98c273/usr/lib/hypnotix/common.py#L179-L181

Although they shouldn't hurt since at least these will be silently skipped later.

Reply:
I agree. The value `15`  seems to be optimal for the prefiltering. Committed.